bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetAutomationHandler.java

```diff
@@ -0,0 +1,427 @@
+                    logger.debug("# {} # target moveTime={}", deviceWhere, moveTime);
+                    if (moveTime > MIN_STEP_TIME_MSEC) {
+                        ScheduledFuture<?> mSch = moveSchedule;
+                        if (mSch != null && !mSch.isDone()) {
+                            // a moveSchedule was already scheduled and is not done... let's cancel the schedule
+                            mSch.cancel(false);
+                            logger.warn("# {} # new XX% requested, old moveSchedule cancelled", deviceWhere);
+                        }
+                        // send a requestFirmwareVersion message to BUS gateways to wake up the CMD connection before
+                        // sending further cmds
+                        OpenWebNetBridgeHandler h = bridgeHandler;
+                        if (h != null && h.isBusGateway()) {
+                            OpenGateway gw = h.gateway;
+                            if (gw != null) {
+                                if (!gw.isCmdConnectionReady()) {
+                                    logger.debug("# {} # waking-up CMD connection...", deviceWhere);
+                                    send(GatewayMgmt.requestFirmwareVersion());
+                                }
+                            }
+                        }
+                        // REMINDER: start the schedule BEFORE sending the command, because the synch command waits for
+                        // ACK and can take some 300ms
+                        logger.debug("# {} # Starting schedule...", deviceWhere);
+                        moveSchedule = scheduler.schedule(() -> {
+                            logger.debug("# {} # moveSchedule expired, sending STOP...", deviceWhere);
+                            try {
+                                sendHighPriority(Automation.requestStop(w));
+                            } catch (OWNException ex) {
+                                logger.warn("Exception while sending request for command {}: {}", command,
+                                        ex.getMessage());
+                            }
+                        }, moveTime, TimeUnit.MILLISECONDS);
+                        logger.debug("# {} # ...schedule started, now sending highPriority command...", deviceWhere);
+                        if (percent < positionEstimation) {
+                            sendHighPriority(Automation.requestMoveUp(w));
+                        } else {
+                            sendHighPriority(Automation.requestMoveDown(w));
+                        }
+                        logger.debug("# {} # ...gateway.sendHighPriority() returned", deviceWhere);
+                    } else {
+                        logger.debug("# {} # moveTime <= MIN_STEP_TIME_MSEC ---> do nothing", deviceWhere);
+                    }
+                } else {
+                    logger.info(
```

Comment:
Shouldn't this set the status to offline? The user should not have to look in the log to find out about the problem reported here,

Reply:
In fact no: the Thing can still be used for UP/DOWN commands, but a POSITION% command cannot be executed until shutterRun config param is set correctly (>=1000) or the position of the shutter becomes known, for example by rolling it fully up/down. So I would not put the Thing offline, but just give using INFO this information so the user can check it.